src/components/draggable/draggable.pw.tsx

```diff
@@ -0,0 +1,123 @@
+import { expect, test } from "@playwright/experimental-ct-react17";
+import React from "react";
+import {
+  draggableItem,
+  draggableItemByPosition,
+} from "../../../playwright/components/draggable";
+import { checkAccessibility } from "../../../playwright/support/helper";
+
+import {
+  DraggableCustom,
+  DraggableDefault,
+  DraggableDifferentContainers,
+} from "../../../src/components/draggable/components.test-pw";
+
+test.describe("Check functionality for Draggable component", () => {
+  ([
+    ["One", 1],
+    ["One", 2],
+    ["One", 3],
+    ["One", 4],
+    ["Two", 3],
+    ["Three", 1],
+    ["Four", 2],
+  ] as const).forEach(([record, destinationId]) => {
+    test(`should drag ${record} and re-order DraggableItem to the ${destinationId} position`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<DraggableCustom />);
+
+      const draggableItemElementRecord = await draggableItem(page, record);
+      const draggableItemByPositionElementDestinationId = await draggableItemByPosition(
+        page,
+        destinationId
+      );
+      await draggableItemElementRecord.dragTo(
+        draggableItemByPositionElementDestinationId
```

Comment:
There seems to be an issue here where the 1st and 2nd positions are mixed up. Each test which should drag the item to position 1 drags it to position 2 and vice versa. Or at least that's how it appears to me when I run in browser!

Reply:
Well what a bug that was. It existed already in Cypress and was caused by [this](https://github.com/Sage/carbon/blob/master/cypress/locators/draggable/index.js#L21). I've decided to re-write it to be a bit more robust.